Xamarin.Forms.Platform.Android/Renderers/ScrollViewRenderer.cs

```diff
@@ -424,5 +430,17 @@ void UpdateOrientation()
 				}
 			}
 		}
+
+		void UpdateHorizontalScrollBarVisibility()
+		{
+			if (_hScrollView != null)
```

Comment:
Can use ?.

Reply:
Fine with me, doing the if seemed to fit in with the rest of the code style better me.

Reply:
Yeah, that seems fine if is matches current style